ciao-controller/tenants.go

```diff
@@ -85,8 +85,8 @@ func (c *controller) CreateTenant(tenantID string, config types.TenantConfig) (t
 	if config.SubnetBits == 0 {
```

Comment:
The comment on the link above should be changed as well, e.g.,

// SubnetBits must be between 4 and 30